src/components/customRenderers/MultiSelect/MultiSelect.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react';
+import { withJsonFormsControlProps } from '@jsonforms/react';
+import { ErrorMessage } from '@amsterdam/asc-ui';
+import { CreatableSelect as Creatable } from '@amsterdam/bmi-component-library';
+import { ControlProps, JsonSchema7 } from '@jsonforms/core';
+import useCustomControl from '../../../hooks/useCustomControl';
+import AccessibleLabel from '../AccessibleLabel/AccessibleLabel';
+
+type SelectOptionType = {
+	value: string;
+	label: string;
+};
+
+function isInsideModal(id: string): boolean {
+	return !!document.getElementById(id)?.closest('[role=dialog]');
+}
+
+function convertValueToOptions(arr: string[]): SelectOptionType[] {
+	return arr.map(option => ({
+		value: option,
+		label: option
+	}))
+}
+const MultiSelect = (props: ControlProps) => {
+	const {
+		data: value = [],
+		path,
+		label,
+		errors,
+		schema: { items: options = [] },
+	} = props;
+
+	// @ts-ignore
```

Comment:
Waar zeurt hij over? En moeten we dit misschien niet op een later moment verbeteren?